mlblocks/mlpipeline.py

```diff
@@ -224,6 +263,39 @@ def _get_str_output(self, output):
 
         raise ValueError('Invalid Output Specification: {}'.format(output))
 
+    def get_inputs(self, fit=True):
+        """Get a dictionary mapping all input variable names required by the
```

Comment:
The docstring looks great but the first line should be shorter, so it fits in a single line.

Then, after the blank line, you can add as many paragraphs as needed.

```python
def get_inputs(self, fit=True):
    """Get a relation of all the input variables required by this pipeline.

    The result is a dictionary that maps each variable name with their
    specified information.
    Optionally include the fit arguments.

    Args:
        ...
    """